includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+class DesignSystemSharedLinks {
+
+	/** @var DesignSystemSharedLinks */
+	private static $instance;
+
+	private function __construct() {
+	}
+
+	public static function getInstance() {
+		if ( !isset( static::$instance ) ) {
+			static::$instance = new DesignSystemSharedLinks();
+		}
+		return static::$instance;
+	}
+
+	public function setHrefs( $hrefs ) {
+		$this->hrefs = $hrefs;
+		return $this;
+	}
+
+	/**
+	 * @param $name string key for href
+	 * @param $lang string two letter language code
+	 * @return string full URL, in case of lang specific URL missing, default one is returned
+	 */
+	public function getHref( $name, $lang ) {
+		return $this->hrefs[ $lang ][ $name ] ?? $this->hrefs[ 'default' ][ $name ];
+	}
+
+	private $hrefs = [
+		'default' => [
+			'fan-communities' => 'http://fandom.wikia.com/explore',
+			'about' => 'http://www.wikia.com/about',
+			'careers' => 'https://careers.wikia.com',
+			'press' => 'http://fandom.wikia.com/press',
+			'contact' => 'http://fandom.wikia.com/contact',
+			'wikia-gives-back' => 'http://www.wikia.com/wikiagivesback',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wikia-gives-back" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)